### ISSUE TITLE
Is it possible to freeze some primary layers?

### ISSUE THREAD

**Issue by kmahsi** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#489]
I'm trying to train a dataset as you said and the problem is that I have such a small dataset and I want to just train the latter layers or just train the softmax layer.
How can I do this?
